setup.py

```diff
@@ -31,7 +31,10 @@ def read(fname):
     license='BSD-3',
     keywords='accounting tax-report portfolio asset-management cryptocurrencies',
     url='https://github.com/rotki/rotki',
-    packages=['rotkehlchen'],
+    packages=find_packages('.'),
```

Comment:
The one thing I am afraid about this change is that pyinstaller also depends on setup.py for creating the binary. I tested it locally and it seemed to work with your changes though.

Reply:
I guess it should work. We just need to check the next release's binaries a bit more thoroughly in case the setup.py changes mess something up in an unexpected way.